mods/cnc/maps/funpark01/map.yaml

```diff
@@ -0,0 +1,544 @@
+		Owner: Neutral
+	Actor90: tc04
+		Location: 31,12
+		Owner: Neutral
+	Actor91: tc04
+		Location: 45,9
+		Owner: Neutral
+	Actor92: tc04
+		Location: 43,16
+		Owner: Neutral
+	Actor93: v01
+		Location: 16,10
+		Owner: Civilian
+		Health: 1
+		Facing: 0
+	Actor94: v02
+		Location: 21,12
+		Owner: Civilian
+		Health: 1
+		Facing: 0
+	Actor95: v03
+		Location: 13,8
+		Owner: Civilian
+		Health: 1
+		Facing: 0
+	Actor96: v04
+		Location: 19,10
+		Owner: Civilian
+		Health: 1
+		Facing: 0
+	Actor97: v05
+		Location: 12,14
+		Owner: Civilian
+		Health: 1
+		Facing: 0
+	Actor98: v06
+		Location: 21,15
+		Owner: Civilian
+		Health: 1
+		Facing: 0
+	Actor99: v06
+		Location: 21,8
+		Owner: Civilian
+		Health: 1
+		Facing: 0
+	Actor100: v07
+		Location: 12,12
+		Owner: Civilian
+		Health: 1
+		Facing: 0
+	Actor101: v07
+		Location: 17,14
+		Owner: Civilian
+		Health: 1
+		Facing: 0
+	Vil04: c4
+		Location: 16,14
+		Owner: Villager
+		Health: 1
+		Facing: 0
+		SubCell: 2
+	Vil03: c3
+		Location: 15,11
+		Owner: Villager
+		Health: 1
+		Facing: 0
+		SubCell: 1
+	Vil01: c2
+		Location: 14,12
+		Owner: Villager
+		Health: 1
+		Facing: 0
+		SubCell: 4
+	Vil02: c1
+		Location: 18,13
+		Owner: Villager
+		Health: 1
+		Facing: 0
+		SubCell: 2
+	WP4: waypoint
+		Location: 13,13
+		Owner: Neutral
+	WP3: waypoint
+		Location: 17,12
+		Owner: Neutral
+	WP2: waypoint
+		Location: 22,14
+		Owner: Neutral
+	WP1: waypoint
+		Location: 42,29
+		Owner: Neutral
+	WP0: waypoint
+		Location: 52,29
+		Owner: Neutral
+	ReinforceA: waypoint
+		Location: 17,52
+		Owner: Neutral
+	ReinforceB: waypoint
+		Location: 15,52
+		Owner: Neutral
+	EntryB: waypoint
+		Location: 15,56
+		Owner: Neutral
+	EntryA: waypoint
+		Location: 17,56
+		Owner: Neutral
+	DinoTrex: trex
+		Location: 52,12
+		Owner: Dinosaur
+	DinoSteg: steg
+		Location: 49,22
+		Owner: Dinosaur
+	DinoTric: tric
+		Location: 53,29
+		Owner: Dinosaur
+	CameraStart: waypoint
+		Location: 16,50
+		Owner: Neutral
+
+Smudges:
+
+Rules:
+	Player:
+		-ConquestVictoryConditions:
+		MissionObjectives:
+			EarlyGameOver: true
+	World:
+		-CrateSpawner:
+		-SpawnMPUnits:
+		-MPStartLocations:
+		LuaScript:
+			Scripts: scj01ea.lua
+		ObjectivesPanel:
+			PanelName: MISSION_OBJECTIVES
+		PlayMusicOnMapLoad:
+			Music: j1
+			Loop: false
+	^Bridge:
+		Invulnerable:
+	^Infantry:
+		MustBeDestroyed:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Vehicle:
+		MustBeDestroyed:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^CivInfantry:
+		-ActorLostNotification:
+	^DINO:
+		MustBeDestroyed:
+		Buildable:
+			Owner: dino
+		Tooltip:
+			ShowOwnerRow: false
+	OLDLST:
+		Inherits: LST
+		-WithRoof:
+		-Selectable:
+		RejectsOrders:
+	TREX:
+		Health:
+			HP: 750
+		Mobile:
+			Speed: 34
+		AutoTarget:
+			ScanRadius: 1
+	TRIC:
+		Health:
+			HP: 700
+		Mobile:
+			Speed: 18
+		AutoTarget:
+			ScanRadius: 5
+	STEG:
+		Health:
+			HP: 600
+		Mobile:
+			Speed: 32
+		AutoTarget:
+			ScanRadius: 4
```

Comment:
Isn't ```Radius: 4``` the [default](https://github.com/OpenRA/OpenRA/blob/bleed/mods/cnc/rules/defaults.yaml#L312)?